### ISSUE TITLE
Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985)

### ISSUE THREAD

**Issue by thepaypay** [MetaMask/metamask-extension#9510]
I deployed a contract/collection through Rarible but when i try to mint the NFT i get this error.

**Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985)**

I have tried updating firmware, app, enabling contracts and reinstalling Metamask. Worth noting everything else works (uniswap,compound,yearn) but not minting these NFT's. I have tried using my own contract and the default Rariable contract but i get the same error. Weird part is I minted one 2 days ago and it worked. Obviously i have googled the error code but the only suggestions are things i have already tried; update firmware, update app, change usb port/cord, change browser ect. Not sure what else i can do or if my ledger nano S is somehow faulty? Any help would be greatly appreciated if anyone has any ideas. Thank you in advance!

**Screenshots**
![Screenshot (171)_LI](https://user-images.githubusercontent.com/72546996/95435742-cfc2c000-0907-11eb-814b-429c07f9fbd4.jpg)

 - OS: [Windows 10]
 - Hardware Wallet [Ledger Nano S Firmware version 1.6.1]
 - Browser [Chrome Version 86.0.4240.75 (Official Build) (64-bit)
 - MetaMask Version [8.0.10]

**Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985)**

**Comment by maciejka** [comment]
@thepaypay did you manage to solve the problem?

**Comment by thepaypay** [comment]
No i did not. Granted i kinda gave up and haven't tried in a couple days. I
ordered a new nano ledger s that i will try later tonight. I will let you
know if that solves the problem.

**Comment by thepaypay** [comment]
So update on the brand new ledger nano s.
Still not working giving me the same error message. No idea what to do
next, maybe i'm just not meant to make NFT's?
New one works with everything but Rarible. Tried it with my own
collection/contract and the default rarible contract.

**Comment by maciejka** [comment]
@thepaypay It seems that the problem is that library that metamask uses to communicate with ledger is outdated. You should ping mm.

**Comment by jacobcantele** [comment]
This is actually an issue with how Windows 10 handles u2f security. We're working on a new solution together with the Ledger team that will replace their current implementation and provide a long term stable resolution.

**Comment by jacobcantele** [comment]
related to #8120

**Comment by jacobcantele** [comment]
This issue will be resolved by #9826

**Comment by thepaypay** [comment]
Thank you for your response. I didnt see many people having this problem so
i appreciate the support! I shal test it out later tonight.

**Comment by JH-node** [comment]
This is not just an issue with windows getting the same error message on Mac, linux and windows

Unable to complete transaction: Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985)

**Comment by alexandernacho** [comment]
Having the same issue on Mac

**Comment by CryptoPassionF** [comment]
Having the same issue with crypto.com defi swap
Error: Transport Status Error: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985)

Not able to withdraw important amount of crypto due to that. Please put some urgency on that issue....

**Comment by danfinlay** [comment]
[According to Ledger,](https://www.reddit.com/r/ledgerwallet/comments/j8bfng/when_trying_to_mint_a_nft_on_rarible_i_keep/g8aotp7?utm_source=share&utm_medium=web2x&context=3) this error was fixed in a newer version of their firmware. We are currently moving our Ledger support to use the Ledger Live app as a bridge, which should solve this issue.

**Comment by Derek-node** [comment]
> [According to Ledger,](https://www.reddit.com/r/ledgerwallet/comments/j8bfng/when_trying_to_mint_a_nft_on_rarible_i_keep/g8aotp7?utm_source=share&utm_medium=web2x&context=3) this error was fixed in a newer version of their bridge library. We are currently moving our Ledger support to use the Ledger Live app as a bridge, which should solve this issue.

Do you have an ETA on when this will be fixed?

**Comment by sebastianzolg** [comment]
I do have the same issue with MEW, and I'm willing to switch to MetaMask. Is there any news on the ETA? Thanks.

**Comment by BlankerL** [comment]
Same error when interacting with OpenSea, glad I have a Trezor as backup device...

**Comment by Derek-node** [comment]
Do we have an update on this?

**Comment by roxdanila** [comment]
@danfinlay friendly ping - do you have an ETA on this release? One of the core Nexus Mutual flows is also impacted, and we have to plan accordingly. Thanks!

**Comment by Alexintosh** [comment]
This a real issue for me as well. Was not able to interact with Aragon apps since a while.

**Comment by johnnieskywalker** [comment]
have the same problem on Ubuntu with Metamask and Ledger on 1inch exchange

**Comment by 3cl1ps** [comment]
having this issue on ubuntu, metamask and nanox. trying to interract with rencrv contract.

**Comment by Gudahtt** [comment]
Is anybody still experiencing this issue? We updated to v5.22.0 of the Ledger library on Jan. 21st, which likely solved this problem.

**Comment by darkwing** [comment]
@3cl1ps @thepaypay @maciejka @Alexintosh @johnnieskywalker We've upgraded our library to 5.43 as of yesterday -- are you still experiencing this issue?

**Comment by BrillerJong** [comment]
Hello, I'm having the same problem... I bought some $AVAX on Binance and transferred them on https://wallet.avax.network.
Then I switched them from the X-Chain to the C-Chain and send all my $AVAX on my metamask address who is connected to my ledger. So they are now on my metamask address and when i open the Pangolin App (https://app.pangolin.exchange/#/swap) for example i take $avax and $png to swap, I need to validate the confirmation in my metamask like always / like on other chains (BSC and ether) then I receive the transaction on my ledger, I validate the transaction like always BUT then I receive the following message : Swap failed: Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985) . I tried over ten times, always the same result. So i decided to test with an other pair. I took the avax/usdt pair and tried with 10$ (0,6x avax). With this pair I didn't received the fail message instead the transaction got accepted, BUT like you see in the screenshot the transaction is still "pending" it's now over +- 24h... And also if I want to check the transaction, this site tells me to wait another 30 secs... :  https://cchain.explorer.avax.network/tx/0x4926fc6e9ffa985aa0106f06b3c8f973afb0812da92ca4daab0043d7d92335a7

I really dont know what to do anymore... I don't know where the problem come from... blockchain? metamask? ledger? pc? i don't know.... I'M REALLY GRATEFUL, if someone can help me!

Thank you in advance,
<img width="1435" alt="Screenshot 2021-03-08 at 19 25 33" src="https://user-images.githubusercontent.com/80281376/110365447-590c2900-8045-11eb-83f4-a5447e28b20a.png">
<img width="1437" alt="Screenshot 2021-03-08 at 19 35 36" src="https://user-images.githubusercontent.com/80281376/110365570-79d47e80-8045-11eb-84df-551cc06b1885.png">

**Comment by BrillerJong** [comment]
Hello i got the same problem on the avalanche c-chain. With this video I fixed the problem: https://youtu.be/-V8HNp46Hk4

**Comment by 0xpicklerickle** [comment]
I just encountered the same issue using it on avalanche's c-chain. However, it's specific to making a swap tx of more than 10AVAX on any of the exchanges (pangolin, zero). Anything less would not trigger the error. I can send and interact with other contracts without a problem at the moment. Could it be the contract data size?

**Comment by jacobcantele** [comment]
It looks like this issue was resolved for the vast majority of chains, but persists on Avalanche. There may be some chain specific issues with Avalanche and Ledger compatibility on the Ledger side.  We've reached out to the Ledger team for feedback.

**Comment by dmjeremy** [comment]
Hello, Since few days now i'm having this same described error but with AAVE on Polygon. the issue appears only when i want to claim the farmed Matic, Otherewise everything else is working correctly on the AAVE plateform.

Can you advise me what do i need to do to get this issue fixed ?

I'm using Ledger Nano S and i'm with Mac

**Comment by derasd** [comment]
Same here with claiming matic on aave polygon.

**Comment by planc765** [comment]
Also unable to claim Matic on Aave/Polygon - bit more info:

- Worked fine up until yesterday (May 1st)
- Can repro on both Nano S and Nano X
- Fresh Metamask install on Microsoft Edge has same problem
- Tried with a different laptop and also same problem
- Tried with an older Metamask build - 9.2.1 and still same problem

Please can this get looked at as a priority as we don't seem to have a workaround at the moment?

**Comment by derasd** [comment]
Interestingly, increasing the gas price to 5 GWEI workes for me.

**Comment by planc765** [comment]
Thanks! that worked for me too

**Comment by wmhilton** [comment]
Mmmm I'm running into this error on https://unstoppabledomains.com after updating my Ledger Nano S to the newest firmware 2.0. :/ I'll try downgrading the firmware if I can figure out how.

**Comment by kennyd28** [comment]
Hey Guys, 

I am stuck with my Checkpoint Arrived stage, where my USDC is burnt and now ready to move onto Ethereum (Into my Ledger Nano X via my Metasmask). I click on Continue and follow all the "Approve buttons" clicking on my Ledger till the "Approve transaction, accept and send" I click that on my ledger, But this is the error that pops up on Metamask, as you guys said Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985). Up until last it was working like butter. But today i updated my Ethereum app on my Ledger to Version 1.7.4 and now it just wont work. 

What do i do ? Any guidance would be much appreciated. Thank You 

![Picture2](https://user-images.githubusercontent.com/83816192/117436120-3d8da380-af4c-11eb-8c31-a78865861141.png)

**Comment by ebonhawk335** [comment]
Hello

Je suis bloqué a l'étape Checkpoint Arrived sur le Bridge Matic , j'ai mes jeton xSDT qui sont gravé et prêt à passer à Ethereum avec ma Ledger Nano X via  Metasmask Je clique sur Continuer et je valide à plusieurs reprise la transaction , jusqu'à 50x mais il ne me propose jamais de send du coup je peut continuer indéfiniment comme ca et rien ne se passe.
Voici le txn de la transaction 0x75fb65e65f7e692e3df5ca80ab0fdf27973fd0d630c91e86427a379d7aedda3b
Est il possible de revenir en arrière?

Pouvez vous m'aider SVP

**Comment by ebonhawk335** [comment]
J'ai trouvé la solution il faut aller dans setting debug data sur la ledger nano x et mettre NOT Displayed et sa fonctionne

**Comment by kennyd28** [comment]
i tried that and it still did not work. My USDC is stuck and will not withdraw. Same issue  Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985) 

Anyone else who can help ? 

> J'ai trouvé la solution il faut aller dans setting debug data sur la ledger nano x et mettre NOT Displayed et sa fonctionne

**Comment by Bropps** [comment]
I have the same error message when trying to create a Maker vault, please help.

**Comment by this-username-is-taken** [comment]
Same issue here

**Comment by Zerocoooler** [comment]
same issue 12 hours ago from Matic Bridge

**Comment by kennyd28** [comment]
So what do we do guys to solve this problem ? I have emailed Ledger and Metamask Support and still waiting on a solution

**Comment by Zerocoooler** [comment]
version 1.7.7 just released and DID NOT solve the issue.

**Comment by kennyd28** [comment]
Yes, i updated to 1.7.7 ETH App as well, and still does not solve the problem. I am so frustrated. Are you using a windows 10 pc ?

**Comment by Zerocoooler** [comment]
correct W10

**Comment by darkwing** [comment]
There's nothing we can do to prevent U2F issues with Ledger devices, but there's good news!  MetaMask 9.6.0 will provide an experimental new Ledger Live bridge, which you will be able to toggle via Advanced settings.

**Comment by kennyd28** [comment]
When does that Metamask 9.6.0 Update come out Darkwing ? 

> There's nothing we can do to prevent U2F issues with Ledger devices, but there's good news! MetaMask 9.6.0 will provide an experimental new Ledger Live bridge, which you will be able to toggle via Advanced settings.

**Comment by this-username-is-taken** [comment]
I'm having this issue on a Mac so definitely not a Windows problem. I've been thinking about initializing a non-Ledger wallet using the mnemonic, make the transaction, and migrate to a new wallet. Probably will do that as a last resort

**Comment by Deepcryptodive** [comment]
> Interestingly, increasing the gas price to 5 GWEI workes for me.

This is the only thing that worked for me! Was already afraid I somehow bricked my Ledger.

Please look into this Metamask team!

**Comment by this-username-is-taken** [comment]
Increasing (or decreasing) the gas price did not work for me.

But I did discover something that might be useful:

![image](https://user-images.githubusercontent.com/2354636/118943696-b4417e00-b908-11eb-9d1d-add30ca15527.png)

![image](https://user-images.githubusercontent.com/2354636/118943720-bc012280-b908-11eb-8776-4b36c6122bd3.png)

This is the data shown on my ledger when confirming a Matic bridge withdrawal transaction of AAVE token. Both the token and the amount are incorrect. I suspect the data that's sent to Ledger is incorrect, hence causing the signature to be rejected.

**Comment by kennyd28** [comment]
On my ledger nano x all the details look accurate when I am trying to withdraw my USDC onto my ledger from the Matic Mainnet, except the Chain ID shows as 0 , and that could be the problem. Very frustrating

**Comment by this-username-is-taken** [comment]
Good point, I see that too and the Ethereum mainnet chain ID should be 1 instead of 0.

**Comment by kennyd28** [comment]
> Good point, I see that too and the Ethereum mainnet chain ID should be 1 instead of 0.

Exactly. It should NOT bbe 0. When i withdraw ETH from the Matic Bridge to my Ledger Wallet, the right Chain ID shows and it goes through smoothly. But with USDC, it shows the wrong Chain ID and wont go through.

**Comment by kennyd28** [comment]
i did reach out to them yes, and they are looking into it. Thanks for Pinging them as well. 

That discord link does not open up for me :( 

> Hmm, in that case, that actually sounds like a Matic problem. I'm pinging their dev:
> 
> https://discord.com/channels/635865020172861441/718850623323504691/844874314850369547
> 
> Seems like their wallet team is taking a look at this.

**Comment by kapoetjos** [comment]
> I'm having this issue on a Mac so definitely not a Windows problem. I've been thinking about initializing a non-Ledger wallet using the Ledger mnemonic, make the transaction, and migrate to a new wallet. Probably will do that as a last resort

I tried to do this, but only metamask is used for the bridging so you can't sign the transaction with other wallets. When importing it into Metamask, you can't import the wallet that you used for the bridge, unless you have the private key.. not sure how to generate this tho. 

I got the same problem as you guys since few days back and support says its on metamask end and they are in direct contact with them, but already a few days passed by.

**Comment by kapoetjos** [comment]
> There's nothing we can do to prevent U2F issues with Ledger devices, but there's good news! MetaMask 9.6.0 will provide an experimental new Ledger Live bridge, which you will be able to toggle via Advanced settings.

So, there is no fix at Metamask end at all? The experimental bridge won't help in this case since the transaction already took place. We are like in the middle of the whole transaction lol.

**Comment by sanchopansa** [comment]
Hi, there! I have the same issue on the Fantom blockchain (since a few days, it was working prior to that, when I last tried).

Device: Ledger Nano X
Firmware: 1.3.0
Ethereum app: 1.7.7
OS: Ubuntu 18.04
MetaMask: 9.5.2

Note that I don't think this is related to U2F, as someone mentioned above because I'm able to successfully submit some types of transactions, while other types fail, e.g. I'm able to stake tokens and claim rewards on the LP farms, but I'm not able to submit any swap transactions across all Fantom DEXes (SpiritSwap, SpookySwap, Sushi). This was definitely not the case a few days ago.

One of the things I'm seeing is that the ChainID for the swapping transactions is set to 0 for some reason, while for Fantom it should be 250. The ChainID is only set to 0 for the swapping transactions, while it's being set correctly for staking/unstaking/claiming rewards, etc.

**Comment by Etropal00** [comment]
> Hey Guys,
> 
> I am stuck with my Checkpoint Arrived stage, where my USDC is burnt and now ready to move onto Ethereum (Into my Ledger Nano X via my Metasmask). I click on Continue and follow all the "Approve buttons" clicking on my Ledger till the "Approve transaction, accept and send" I click that on my ledger, But this is the error that pops up on Metamask, as you guys said Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985). Up until last it was working like butter. But today i updated my Ethereum app on my Ledger to Version 1.7.4 and now it just wont work.
> 
> What do i do ? Any guidance would be much appreciated. Thank You
> 
> ![Picture2](https://user-images.githubusercontent.com/83816192/117436120-3d8da380-af4c-11eb-8c31-a78865861141.png)




Same problem

**Comment by Etropal00** [comment]
same probleme with eth

**Comment by kapoetjos** [comment]
anyone with the problem > update your Ledger ETH app to 1.7.8 version. I think they implemented a new fix in this! It worked after this and the transaction went through.

**Comment by Zerocoooler** [comment]
Solved!

 

Da: kapoetjos ***@***.***> 
Inviato: venerdì 21 maggio 2021 00:15
A: MetaMask/metamask-extension ***@***.***>
Cc: Zerocoooler ***@***.***>; Comment ***@***.***>
Oggetto: Re: [MetaMask/metamask-extension] Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985) (#9510)

 

anyone with the problem > update your Ledger ETH app to 1.7.8 version. I think they implemented a new fix in this! It worked after this and the transaction went through.

**Comment by Zerocoooler** [comment]
Solved!

**Comment by sanchopansa** [comment]
Version 1.7.8 resolves the swap issues on Fantom as well.

**Comment by sanchopansa** [comment]
Actually, I take that back. It only works for small amounts, but if you try to swap larger amounts the ChainID is still being set to 0.

**Comment by kennyd28** [comment]
New ETH 1.7.8 Update solves the problem. YAAAAY for us all haha

**Comment by Etropal00** [comment]
New ETH 1.7.8 Update solves the problem  me too

**Comment by dmjeremy** [comment]
Hello,
I'm experiencing the exact same issue than described above for couple of days, "Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985)". 
i never had this issue before.

it happen only  when i want to claim my matic on AAVE Polygon.

I tried on 2 different computers (Mac and Microsoft), with 2 different ledger Nano S, i tried with different cables, different usb port. all firmware are up to date. I tried as well with different browsers.

Nothing is working.

thank you for any support
